### PR TITLE
Information functions

### PR DESCRIPTION
**Pull request by mmatera**

### CODE REVIEW

File: mathics/doc/doc.py [hunk 1]
```diff
@@ -850,6 +850,21 @@ def __init__(self, doc):
     def __str__(self):
         return '\n'.join(str(item) for item in self.items)
 
+    def text(self, detail_level):
+        # used for introspection
```
Comment (GarkGarcia) [inline mathics/doc/doc.py:854]: We should work in this before we merge the PR

Comment (GarkGarcia) [reply mathics/doc/doc.py:854]: Besides this, I think we're all done in here. II haven't looked at the tests though.

**Comment by GarkGarcia** [comment]
Hey @mmatera, isn't this a duplicate of #618?

**Comment by mmatera** [comment]
It was an attempt to make a rebase of the other version. At this point, I thought it was a bit messy to make a rebase of the other version. The problem is that I will not be able to work on the project during the next few weeks. After that, I will try to fix the incompatibilities and merge both versions in a single pull request.

**Comment by GarkGarcia** [comment]
> It was an attempt to make a rebase of the other version. At this point, I thought it was a bit messy to make a rebase of the other version. 

I see. Could I close the old PR then?

> The problem is that I will not be able to work on the project during the next few weeks. After that, I will try to fix the incompatibilities and merge both versions in a single pull request.

No problem, take your time 😋️

**Pull request closed without merging**